CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+If you're interested in contributing to this project, here are a few ways to do so:
```

Comment:
followup: if rules are not followed we will close the issue/pr. please follow the instructions carefully.